installer/install-esbonio.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -e
+
+python3 -m venv ./venv
+./venv/bin/pip3 install -U pip
+./venv/bin/pip3 install "esbonio[lsp]"
+./venv/bin/pip3 install "pygls==0.9.1"
```

Comment:
It is no longer necessary to install a fixed version of pygls. 😇 <https://github.com/swyddfa/esbonio/issues/147>

Reply:
Thank you! I'll remove this